### PR TITLE
Update the CLI Guide with the Latest Changes

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the CLI Guide with the latest changes.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/using-the-cli-tools.md [hunk 1]
```diff
@@ -51,66 +42,71 @@ These everyday commands are your best friends! They address the very basics of p
 </td>
 </tr>
 <tr>
-<td class="cCommand">test</td>
-<td class="cDescription">Run tests of a particular module or all the modules of a Ballerina project. For more information, see <a href="/learn/how-to-test-ballerina-code">How to Test Ballerina Code</a>.
+<td class="cCommand">run</td>
+<td class="cDescription">Build and run a Ballerina program, a single BAL file, an entire project, or a previously-built program. For more information, see <a href="/swan-lake/learn/running-ballerina-code">Running Ballerina Code</a>.
```
Comment (keizer619) [inline swan-lake/learn/using-the-cli-tools.md:46]: We need to include module as well.

Comment (praneesha) [reply swan-lake/learn/using-the-cli-tools.md:46]: Added.

File: learn/using-the-cli-tools.md [hunk 2]
```diff
@@ -123,15 +119,39 @@ These powerful supporting tools extend Ballerina to various ecosystem technologi
...
+<tr>
 <td class="cCommand">openapi</td>
-<td class="cDescription">This is the OpenAPI (Swagger) stub/skeleton generation tool. For more information, see <a href="/learn/how-to-use-openapi-tools">Ballerina OpenAPI Tools</a>.</td>
+<td class="cDescription">This is the OpenAPI (Swagger) stub/skeleton generation tool. For more information, see <a href="/learn/using-the-openapi-tools">Using the OpenAPI Tools</a>.</td>
 </tr>
 <tr>
-<td class="cCommand">grpc</td>
-<td class="cDescription">This is the gRPC stub/skeleton generation tool. For more information, see <a href="/learn/how-to-generate-code-for-protocol-buffers">How to generate Ballerina code for Protocol Buffer Definition</a>.</td>
+<td class="cCommand">version</td>
+<td class="cDescription">Tells you the version of Ballerina you have installed.</td>
```
Comment (keizer619) [inline learn/using-the-cli-tools.md:131]: @praneesha ; Should we explain this further? That versions shows currently using distribution version and based language specific version. Also it include update tool version as well

Comment (praneesha) [reply learn/using-the-cli-tools.md]: Done.

**Pull request merged**